functions-and-scope.js

```diff
@@ -40,6 +66,18 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 // * Hoe zorgt ik ervoor dat ik alle waardes uit de array kan langslopen, ook als de array wel 100 entries zou bevatten?
 // Log het antwoord in de terminal.
 
+function averageGrade(grades3) {
+    let numberOfGrades = 0;
```

Comment:
`const numberOfGrades = grades3.length;` is beter, want dan hoef je die niet meer op te hogen.

Reply:
Tuurlijk, helemaal niet aan gedacht!